modules/aws/sonar-upgrader/python_upgrader/upgrade/upgrade_status_manager.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import annotations
```

Comment:
I don't understand this change. If this file replaces upgrade_status_service.py where is the deletion of upgrade_status_service.py?
I tried to do a diff in IntelliJ and it's impossible to review.
Please explain the purpose of this change.

Reply:
This should not have been added, deleting.